packages/example/src/ToneJSTesting/ToneJSExample.tsx

```diff
@@ -0,0 +1,79 @@
+import {useEffect, useState} from 'react';
+import {Audio, continueRender, delayRender} from 'remotion';
+
+// Can we use this? :thinking:
+import { getRemotionEnvironment } from '../../../core/dist/get-environment';
+
+import * as Tone from 'tone';
+import toWav from 'audiobuffer-to-wav';
+
+function uint8ToBase64(buffer: ArrayBuffer) {
```

Comment:
This was just a sample I found, it probably needs to be extracted and maybe ensure this is the fastest method.